docs/source/markdown/podman-pod-rm.1.md

```diff
@@ -49,6 +49,15 @@ podman pod rm -fa
 
 podman pod rm --pod-id-file /path/to/id/file
 
+## Exit Status
+  **0**   All specified pods removed
+
+  **1**   One of the specified pods did not exist, and no other failures
+
+  **2**   One of the specified pods is attached to a container
+
+  **125** The command fails for a reason other than pod did not exist
```

Comment:
```suggestion
  **125** The command fails for a reason other than pod did not exist or network in use
```

Reply:
This is podman pod rm, nothing to do with Netoworks, I believe.

Reply:
Can we compromise on "...for a reason other than those listed above"?